templates/install/kubernetes/addons/external-lb.yaml

```diff
@@ -1,73 +1,16 @@
 apiVersion: v1
-items:
-- apiVersion: v1
-  kind: Service
-  metadata:
-    annotations: {}
-    labels:
-      app: enmasse
-    name: address-controller-external
-  spec:
-    ports:
-    - name: https
-      port: 443
-      protocol: TCP
-      targetPort: https
-    selector:
-      name: address-controller
-    type: LoadBalancer
-- apiVersion: v1
-  kind: Service
-  metadata:
-    labels:
-      app: enmasse
-    name: messaging-external
-  spec:
-    ports:
-    - name: amqp
-      port: 5672
-      protocol: TCP
-      targetPort: 5672
-    - name: amqps
-      port: 5671
-      protocol: TCP
-      targetPort: 5671
-    selector:
-      capability: router
-    type: LoadBalancer
-- apiVersion: v1
-  kind: Service
-  metadata:
-    annotations: {}
-    labels:
-      app: enmasse
-    name: mqtt-external
-  spec:
-    ports:
-    - name: mqtt
-      port: 1883
-      protocol: TCP
-      targetPort: 1883
-    - name: secure-mqtt
-      port: 8883
-      protocol: TCP
-      targetPort: 8883
-    selector:
-      name: mqtt-gateway
-    type: LoadBalancer
-- apiVersion: v1
-  kind: Service
-  metadata:
-    labels:
-      app: enmasse
-    name: console-external
-  spec:
-    ports:
-    - name: console-ws
-      port: 56720
-    - name: console-http
-      port: 8080
-    selector:
-      name: admin
-    type: LoadBalancer
-kind: List
+kind: Service
+metadata:
+  annotations: {}
+  labels:
+    app: enmasse
+  name: restapi-external
+spec:
+  ports:
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: https
+  selector:
+    name: address-controller
+  type: LoadBalancer
```

Comment:
Why here we have the service only for the address controller and not for all the other services (messaging, mqtt, ... ) ?

Reply:
@ppatierno They are created based on the address space endpoints. By default, it creates the services for messaging, mqtt and console unless otherwise restricted. This allows different address spaces to get their own loadbalancer endpoints.

From the user side of things, the 'default' singletenant deployment will look like before, but with the endpoints as loadbalancer services instead of ingress. With minikube, it actually exposes loadbalancer services as nodeports, so you are able to more easily the services as well.

Reply:
ok got it, thanks !

Reply:
@ppatierno Relevant code is here: https://github.com/EnMasseProject/enmasse/pull/560/files#diff-f843557b1ea4676aeecc78f15d6db723R148

Reply:
Yes I miss the point that the address controller is in charge to deploy all the other stuff so it has to create the services in the address space and then exposing them with loadbalancer. Thanks @lulf !